src/main/java/pl/nogacz/checkers/application/Design.java

```diff
@@ -64,6 +69,56 @@ public static void addLightMove(Coordinates coordinates) {
     }
 
     public static void removePawn(Coordinates coordinates) {
-        gridPane.getChildren().removeIf(node -> GridPane.getColumnIndex(node) == coordinates.getX() && GridPane.getRowIndex(node) == coordinates.getY());
+       gridPane.getChildren().removeIf(node -> GridPane.getColumnIndex(node) == coordinates.getX() && GridPane.getRowIndex(node) == coordinates.getY());
+    }
+
+    public static void changeScore(HashMap<Coordinates, PawnClass> board){
+        int computerScore = 0;
+        int playerScore = 0;
+        for(Map.Entry<Coordinates, PawnClass> entry : board.entrySet()) {
+            if(entry.getValue().getColor() == PawnColor.BLACK){
+                computerScore++;
+            }else if(entry.getValue().getColor() == PawnColor.WHITE){
+                playerScore++;
+            }
+        }
+        generateHealthBar(playerScore, computerScore);
+    }
+
+    public static void generateHealthBar(int playerScore, int computerScore){
+        //delete old bars
+        removePawn(new Coordinates(0,8));
+        removePawn(new Coordinates(0,8));
+
+        //decide bar lengths according to the current score
+        int total = computerScore+playerScore;

```

Comment:
fix spacing

`computerScore+playerScore` => `computerScore + playerScore`

Bu dosyada eklediğin diğer satılara da benzer değişikliği uygula lütfen.